getting-started/002-search.md

```diff
@@ -37,20 +35,23 @@ If you clicked on the query link above, you probably saw some cool **GeoJSON**,
 * YMCA, Jefferson, OH
 * YMCA, Belleville, IL
 
-Note that the results are spread out throughout the world. Since we haven't told the service anything about our current location or any other geographic context.
+Spelling matters, but not capitalization when performing a query with Mapzen Search. You can type `ymca`, `YMCA`, or even `yMcA`. See for yourself by comparing the results of the previous search to the following:
+
+> [/v1/search?api_key={YOUR-KEY}&___text=yMcA___](https://search.mapzen.com/v1/search?api_key={YOUR_API_KEY}&text=yMcA)
+
+Note that the results are spread out throughout the world because you have not given your current location or provided any other geographic context in which to search.
 
-## Narrowing your Search...
+## Narrow your search
 
-Sometimes it's necessary to limit the search to a portion of the world. This can be useful if you're looking for places in a particular region, or country, or only want to look in the immediate viscinity of a user with a known location. Different usecases call for different specifications of this bounding region. We currently support three types: **rectangle**, **circle**, and **country**.
+If you are looking for places in a particular region, or country, or only want to look in the immediate vicinity of a user with a known location, you can narrow your search to an area. There are different ways of including a region in your query. Mapzen Search currently supports three types: country, rectangle, and country.
```

Comment:
D'oh. Good catch. Copy/paste error from me trying to reorg it to match the order of the sections below.